Formula/poppler.rb

```diff
@@ -58,6 +58,9 @@ class Poppler < Formula
   def install
     ENV.cxx11
 
+    # removes /usr/include from CFLAGS (not clear why)
+    ENV["PKG_CONFIG_SYSTEM_INCLUDE_PATH"] = "/usr/include" if MacOS.version < :mojave
```

Comment:
Maybe it wants to try to check the SDK? No idea why, though, as there won't be any `.pc` files to read there.